RELEASES.md

```diff
@@ -1,3 +1,123 @@
+Version 1.81.0 (2024-09-05)
+==========================
+
+<a id="1.81.0-Language"></a>
+
+Language
+--------
+
+- [Abort on uncaught panics in `extern "C"` functions.](https://github.com/rust-lang/rust/pull/116088/)
+- [Fix ambiguous cases of multiple `&` in elided self lifetimes.](https://github.com/rust-lang/rust/pull/117967/)
+- [Stabilize `#[expect]` for lints (RFC 2383),](https://github.com/rust-lang/rust/pull/120924/) like `#[allow]` with a warning if the lint is _not_ fulfilled.
+- [Edition 2024: Make `!` fall back to `!` instead of `()`.](https://github.com/rust-lang/rust/pull/123508/)
```

Comment:
do we want to put this here? it won't affect users.

Reply:
It made the list because of its FCP, but I'd be fine with dropping it.

Reply:
Yeah, I think we should drop e2024 things. Those will get "announced" as part of the edition itself (edition guide etc). Until then they're not really stable either.